src/acceleration/impl/ResidualSumPreconditioner.cpp

```diff
@@ -45,20 +45,31 @@ void ResidualSumPreconditioner::_update_(bool                   timestepComplete
       norms[k] = std::sqrt(norms[k]);
     }
     sum = std::sqrt(sum);
-    PRECICE_CHECK(not math::equals(sum, 0.0), "All residual sub-vectors in the residual-sum preconditioner are numerically zero. "
-                                              "Your simulation probably got unstable, e.g. produces NAN values.");
+    if (math::equals(sum, 0.0)) {
+      PRECICE_WARN("All residual sub-vectors in the residual-sum preconditioner are numerically zero ( sum = " << sum << "). This indicates that the data values exchanged between two succesive iterations did not change."
+                                                                                                                         " The simulation may be unstable, e.g. produces NAN values. Please check the data values exchanged"
+                                                                                                                         " between the solvers is not identical between iterations.");
+    }
 
     for (size_t k = 0; k < _subVectorSizes.size(); k++) {
       _residualSum[k] += norms[k] / sum;
-      PRECICE_CHECK(not math::equals(_residualSum[k], 0.0), "A sub-vector in the residual-sum preconditioner became numerically zero. "
-                                                            "Thus, the preconditioner is no longer stable. Please try the value preconditioner instead.");
+      if (math::equals(_residualSum[k], 0.0)) {
+        PRECICE_WARN("A sub-vector in the residual-sum preconditioner became numerically zero ( residualSum = " << _residualSum[k] << ") . If this occured directly after the second iteration, check that the initial-relaxation"
+                                                                                                                                      " factor is not equal to 1.0 if the coupling values of one solver"
+                                                                                                                                      " is zero in the first iteration."
+                                                                                                                                      " Please try freezing the preconditioner weights when using residual-sum"
+                                                                                                                                      " or use the value preconditioner instead.");
```

Comment:
Same as above, we need to tell the user what is happening. And we should weaken the statements, since we already have a solution now. Especially the freezing makes no sense then.
Please don't use variable names such as `residualSum`. Those are not understandable to users not reading the source code.